### ISSUE TITLE
App not compatible with Intel?

### ISSUE THREAD

**Issue by gb160** [bilbofroggins/KarabinerKeyboard#22]
Hi, when I extract the zip the .app file has a strike through it and I get the message:

You can’t open the application “KarabinerKeyboard” because this application is not supported on this Mac.

..when I try to open it.
MacBook Pro 2018 Intel i5

**Comment by bilbofroggins** [comment]
Yeah sorry, I was putting this off because I don't have an intel mac and that means I need rosetta and a dual homebrew/python/etc setup in order to bundle the app with the x86 version of python. I'll get around to this sometime soon but in the meantime, if you've got python3 on your machine, you might be able to just run the bundle steps yourself:

pip3 install poetry
poetry shell
poetry install
poetry run bundle-app

The last part of bundle-app will fail because it's the part that notarizes with Apple but it should still build you an executable in the dist folder

**Comment by gb160** [comment]
no problem mate I'll try that method.
